data/campaigns/Eastern_Invasion/_main.cfg

```diff
@@ -25,20 +26,18 @@
 [binary_path]
     path=data/campaigns/Eastern_Invasion
 [/binary_path]
-{campaigns/Eastern_Invasion/terrain.cfg}
-{campaigns/Eastern_Invasion/terrain-graphics.cfg}
 #endif
 #ifdef EDITOR
 [binary_path]
-    path=data/campaigns/Eastern_Invasion
+    path=data/resources/Urban_Jungle
 [/binary_path]
+{resources/Urban_Jungle/terrain.cfg}
+{resources/Urban_Jungle/terrain-graphics.cfg}
 [editor_group]
-    id=eastern_invasion
+    id=urban_jungle
     name= _ "Eastern Invasion"
```

Comment:
In addition to changing the ID, the name needs to be changed here as well.

Reply:
Done, and moved to the toplevel `_main.cfg`